### ISSUE TITLE
Do the handles show on "Design your field layout" sections?

### ISSUE THREAD

**Issue by kevadamson** [mmikkel/CpFieldLinks-Craft#6]
i.e. When adding fields to entry types in sections?

**Comment by mmikkel** [comment]
No, sorry. I agree it'd be useful, though I'm not really sure how to fit into the layout for the Field Layout Designer.

In any case, good feature request – thanks!

**Comment by kevadamson** [comment]
@mmikkel Excuse my ignorance at what might be possible with a plugin, but I was thinking something like this?:

![handle1](https://user-images.githubusercontent.com/1021744/34833387-27241640-f6e7-11e7-9a3d-84320664b2cc.png)

`<small style="display: block;text-transform: none;font-size: 11px;opacity: 0.5;">headingHandle</small>`

Or perhaps just a title tag:

![handle2](https://user-images.githubusercontent.com/1021744/34833400-2fb2ac90-f6e7-11e7-89fa-bfd7dbf42d97.png)

 `<span title="headingHandle">Heading</span>`

**Comment by mmikkel** [comment]
Yeah, that could work. I'm not crazy about how much your suggestion would add to the height for for each field; if you have a lot of fields then the scroll-a-mathon of the Field Layout Designer would be even worse. Perhaps a better suggestion would be to make the name clickable; when you click it the name is replaced w/ the handle?

**Comment by kevadamson** [comment]
@mmikkel Sounds good. Not too bothered about how it's implemented - always pros and cons, just as long as it's there :)
